app/models/User.ozar.coffee

```diff
@@ -722,6 +722,39 @@ module.exports = class User extends CocoModel
       me.startExperiment('m7', value, probability)
     value
 
+
+  revokePrepaid: (classroom, complete) ->
```

Comment:
there is a function in prepaids.js store revokeLicenses - any chance we can re-use it?